keras_retinanet/utils/colors.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+import matplotlib.colors
```

Comment:
I assume you added the list below to avoid depending on `matplotlib`, but you forgot to remove this import? `numpy` doesn't seem to be necessary anymore either.

Reply:
Very perceptive mister @de-vri-es , I removed those.